src/utils/index.ts

```diff
@@ -5,7 +5,7 @@
 
 import {makeDraggable} from './draggable';
 import {DisplayMode, EnvironmentIndexer, Indexes} from './indexer';
-import {foreachSetting, HTMLSetting, SettingGroup, SettingModificationOrigin} from './settings';
+import {foreachSetting, HTMLSetting, SettingGroup, SettingModificationOrigin, Validator} from './settings';
```

Comment:
Why is this required? The corresponding code does not exists on master

Reply:
On master, npm was throwing a compiler error for this not existing

Reply:
Could to check again? Master is building fine without this: https://github.com/cosmo-epfl/chemiscope/runs/788274793

Reply:
Seems that the error wasn't recreated, so there might have been an artifact from creating the branch. I'll remove this line from the PR.